docs/sdks/js-reference-guide/js-sdk-authenticateOtp.md

```diff
@@ -0,0 +1,64 @@
+---
+title: authenticateOtp
+id: js-reference-authenticateOtp
+description: ''
+slug: /js-reference-authenticateOtp
+keywords:
+ - javascript sdk
+pagination_next: null
+pagination_prev: null
+last_update:
+   date: 07/25/2023
+   author: William May
+draft: false
+doc_type: reference
+displayed_sidebar: sdkSidebar
+---
+
+The **authenticateOtp** function enables an app using the Beyond Identity Javascript SDK to initiate authentication using an OTP. The OTP will be sent to the provided email address.
+
+## Dependencies
+
+The **authenticateOtp** function requires the Beyond Identity Javascript SDK.
+```
+npm install @beyondidentity/bi-sdk-js
+```
+
+## Prerequisites
+
+Before making a call to **authenticateOtp**, you must complete the following prerequisite calls:
+
+1. Import the required types and functions from the SDK
+```javascript
+import {Embedded} from '@beyondidentity/bi-sdk-js';
+```
+
+2. Initialize the SDK
+```javascript
+const embedded = await Embedded.initialize();
+```
+
+3. Use **authenticateOtp** to initiate authentication using an OTP
+```javascript
+await embedded.authenticateOtp(url, email);
+```
+
+## Parameters
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| url | string | Required. The authentication URL of the current transaction. |
```

Comment:
Should we put an example of this url? It's the window.location.href in the HWA which we hit after /authorize but I think the key to this URL is that is /bi-authenticate and that it has a value JWT as a request. 

/bi-authenticate?request=someToken

Reply:
Added.